.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+node_modules
```

Comment:
Please avoid duplicate lines in your code. duplicate lines introduce bugs and unexpected behaviour.
It is better to add a forward slash (/) to the end of the node_modules/ so that it is clear that you are ignoring the node_modules folder